hooks/data_validation.py

```diff
@@ -76,3 +76,39 @@ def sanitize_check_result(self, errors):
         """
 
         raise NotImplementedError()
+
+    def resolve_rules(self, manager, rules):
+        """
+        Helper method to execute the manager resolve rule and then call validate post resolution.
+
+        :param manager: Data validation manager
+        :type manager: <ValidationManager>
+        :param rules: The list of rules to resolve and then validate.
+        :type rules: list<ValidationRule>
+        """
+        manager.resolve_rules(rules)
+        self.post_fix_action(rules)
+        manager.validate_rules(rules)
```

Comment:
For consistency with `resolve_all_rules` we should actually run the `validate` before the `post_fix_action`. The resolve all rules will run validate within the resolve call, when post_validate=True.